portal-test/src/com/liferay/portal/kernel/test/util/PropsValuesReplacer.java

```diff
@@ -12,12 +12,12 @@
  * details.
  */
 
-package com.liferay.adaptive.media.document.library.thumbnails.internal.test.util;
+package com.liferay.portal.kernel.test.util;
 
 import com.liferay.petra.reflect.ReflectionUtil;
 import com.liferay.portal.kernel.util.PropsKeys;
-import com.liferay.portal.util.PropsUtil;
-import com.liferay.portal.util.PropsValues;
+import com.liferay.poshi.core.util.PropsUtil;
```

Comment:
Hi @adolfopa,

This should be `com.liferay.portal.util.PropsUtil. However that won't compile because `portal-test` does not depend on `portal-impl`.

I could not find a way to add that dependency and make it work.

Reply:
Yep, IntelliJ messed up the imports. I'll need to move this anywhere else.

Reply:
I also saw this: https://github.com/liferay/liferay-portal/blob/c6c5cafc50deda9c90357c8ac7d8a193d817eddc/modules/dxp/apps/portal/portal-dao-db/src/test/java/com/liferay/portal/dao/db/SybaseDBTest.java#L34. Seems like an already supported way of doing this.

The problem with that is that it won't work on @Overriden methods from a base test class.